### DISCUSSION TITLE
i18n domain routing does not serve static files

### DISCUSSION THREAD

**Discussion by yhay81** [vercel/next.js#19315] [Help] [answered]
Hello,

I found I cannot serve static files from `public` directory when I use i18n domain routing served from vercel.
Default locale serves correctly but the others returns 404 status.

Here is a simple reproducible example.
https://github.com/yhay81/vercel-i18n-sample

This app is served from vercel via the url below.
https://webv.jp

I mean I cannot get https://en.webv.jp/vercel.svg

Thanks a lot :)

**Comment by ijjk** [comment, marked as answer]
Hi, thanks for the information this does appear to be a bug and I opened an issue [here](https://github.com/vercel/next.js/issues/19324) to track it

**Reply by ijjk** [reply]
@yhay81  this should now be corrected, please re-deploy your application and give it a try!

**Reply by yhay81** [reply]
Thank you for quick support! It works now.
